scripts/reborn_webui_v2_live_qa/google_api_helpers.py

```
@@ -234,20 +234,38 @@ async def _wait_for_google_sheet_marker(
-        )
+        try:
+            last_check = await _google_sheet_contains_marker(
+                access_token=access_token,
+                spreadsheet_id=spreadsheet_id,
+                marker=marker,
+                range_name=range_name,
+            )
+            last_error = None
+        except AssertionError:
+            raise
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)